src/unity/toolkits/recsys/recsys_model_base.cpp

```diff
@@ -163,7 +163,7 @@ void recsys_model_base::setup_and_train(
               << "Warning: Ignoring columns " << columns_ss.str() << ";"
               << std::endl;
           logprogress_stream
-              << "    To use one of these as a target column, set target = <column_name> "
+              << " Specify the column name to be used as target "
```

Comment:
This is conditional, so we should still say "To use one of these as a rating column, ..."